tests/ea/mutation/mutator/test_mutate.py

```diff
@@ -49,7 +49,11 @@ def _test_mutate(mutator, record, mutation_record):
     assert (
         result.get_mutator_name() == mutation_record.get_mutator_name()
     )
+    original_mol = result.get_molecule_record(
+    ).get_molecule().with_canonical_atom_ordering()
```

Comment:
```python
original_mol = (
    result
    .get_molecule_record()
    .get_molecule()
    .with_canonical_ordering()
)
```

Reply:
I think this should not be called original molacule because it imples it is the molecule *before* the mutation took placae. I would just call it `mutated_molecule` and call what you currently call `mutated_mol` `expected_molecule` instead.

Reply:
Sounds good!